### ISSUE TITLE
Fix building placement - allow placement anywhere that should be possible

### ISSUE THREAD

**Issue by RichardJECooke** [JulienNigon/SpellcasterUniversity-Bug-Report#1207]
For example:

![Untitled](https://user-images.githubusercontent.com/5468805/107382514-9fc73b80-6af8-11eb-8794-d077832a823c.jpg)

**Comment by Neelan22** [comment]
I don't know what card you're playing but in this configuration it seems to be a room with a side door. So the only place available is the room ghost appearing in your big red area. The other areas are therefore not accessible.
Without being a great constraint, we have voluntarily put construction rules for every room to make the player think a little bit about how he is going to arrange his castle during the construction. Some rooms require anticipation.

**Comment by RichardJECooke** [comment]
Yes, this aspect of the game is highly unclear. It says when you play the very first mission that some rooms have doors and positioning matters. But to me it looks like all rooms have a back door that people use - left or right side doors can't really be seen from the card and often in the actual room too. Positioning doesn't seem to matter in the least - no room can be unusable and students wander everywhere. 99% of rooms seems to have 3 doors and can be put anywhere.  This 1% constraint seems contrived - I'd either remove it, or explain visually (not through a tutorial) how you intend it to work.

**Comment by Neelan22** [comment]
On each room card, there is a drawing that explains it
